core/src/main/scala/org/apache/mahout/math/algorithms/preprocessing/AsFactor.scala

```diff
@@ -92,6 +115,12 @@ class AsFactorModel(cardinality: Int, factorVec: MahoutVector) extends Preproces
     res
   }
 
+  /**
+    * Inverse transform the input data - ie transform a one-hot encoded DrmLike[K] into a
```

Comment:
I don't know what this method does but that comment doesn't look complete..

Reply:
not sure how I did that... good eye.